measures/ResStockArguments/measure.rb

```diff
@@ -47,10 +47,25 @@ def arguments(model)
     args = OpenStudio::Measure::OSArgumentVector.new
     measure.arguments(model).each do |arg|
       next if Constants.excludes.include? arg.name
+      next if arg.name == 'geometry_cfa'
```

Comment:
@shorowit I think you could instead just append `geometry_cfa` to this list: https://github.com/NREL/resstock/blob/restructure-v3/measures/ResStockArguments/resources/constants.rb#L6-L20.

Reply:
That won't work because `geometry_cfa` is a required argument in the BuildResHPXML measure. I need an identically named argument in this measure that accepts "auto" or a number, since the BuildResHPXML measure argument only allows numbers. If "auto" is supplied, it's converted to a number and passed along.